### PR TITLE
js-objects-and-arrays-arrays solution

### PR DESCRIPTION
**Pull request by ak-coding-ber**

### CODE REVIEW

File: js-objects-and-arrays/js-objects-and-arrays_arrays/index.js [hunk 1]
```diff
@@ -3,51 +3,56 @@ console.clear();
 // EXERCISE 1
 // Modify the array exampleArray so that it contains a number and a string.
 
-const exampleArray = [true];
+const exampleArray = [25, "hello"];
```
Comment (v-aru) [inline js-objects-and-arrays/js-objects-and-arrays_arrays/index.js:6]: I think the approach would be to use some in-built array methods in order to replace the value it currently holds? For example: push/splice combination?

File: js-objects-and-arrays/js-objects-and-arrays_arrays/index.js [hunk 2]
```diff
@@ -3,51 +3,56 @@ console.clear();
...
 
 // EXERCISE 2
 // Nest an array inside array nestedArray. After that nestedArray should contain an array as an element.
 
-const nestedArray = ["example", 10, true];
+const nestedArray = ["example", [25, "hello"], 10, true];
```
Comment (v-aru) [inline js-objects-and-arrays/js-objects-and-arrays_arrays/index.js:11]: Same comment here as above. Suggest to use maybe a push method to insert a new array element into the nestedArray object.

**Review by v-aru — COMMENTED** [review]
Reviewed with suggested changes.

File: js-objects-and-arrays/js-objects-and-arrays_arrays/index.js (hunk 1 above)
Comment (RuslanMirasov) [reply js-objects-and-arrays/js-objects-and-arrays_arrays/index.js:6]: This is the very first assignment, I think it's just on understanding what kinds of data can be added to an array. I agree that it is better to use something like : const exampleArray = [true];
exampleArray[0] = "Some string";
exampleArray.push(100); But it's fine for the first challenge.

**Review by RuslanMirasov — APPROVED** [review]

**Pull request merged**